src/test/java/picard/util/IntervalListScattererTest.java

```diff
@@ -80,6 +85,11 @@ private Testcase(final File file, final int scatterWidth, final IntervalListScat
         Assert.assertEquals(LIST_TO_SCATTER_MANY.getUniqueBaseCount(), 32 * 2, "Wrong unique base count");
         Assert.assertEquals(LIST_TO_SCATTER_MANY.getIntervals().size(), 32, "Wrong unique interval count");
 
+        testCases.add(new Testcase(
+                LARGER_INTERVAL_FILE, 60, IntervalListScatterMode.INTERVAL_COUNT_WITH_DISTRIBUTED_REMAINDER,
+                LARGER_EXPECTED_LISTS
+        ));
+
```

Comment:
add a test where there is no remainder, and make sure all interval_lists end up the same size

Reply:
In a new test, I split the 200 intervals into 20 lists and verified that they each have 10 intervals.